backend/server.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express'); 
+const app = express();
+const port = 8080;
+const fs = require('fs');
```

Comment:
nit: in general, you want to do all of your important (aka `require` statements) at the top of the file, so let's move this to line 2